app/controllers/users_controller.rb

```diff
@@ -0,0 +1,23 @@
+class UsersController < ApplicationController
+  
+
+  def new
```

Comment:
def new and def edit are handled by devise :). You'll only need `def show` in this controller.